lib/Conversion/TorchToTcp/Utils.h

```diff
@@ -22,5 +22,21 @@ Value broadcastShapeInLeadingDims(ConversionPatternRewriter &rewriter,
 Value broadcastInLeadingDimsToMatchShape(ConversionPatternRewriter &rewriter,
                                          Value input, Value target);
 
+Value broadcast0DToNDAndMatchShape(ConversionPatternRewriter &rewriter,
+                                   Value input, Value target);
+
+Value broadcast1DToNDAndMatchShape(ConversionPatternRewriter &rewriter,
+                                   Value input, Value target,
+                                   int64_t axisInOutput);
+
+template <typename T>
+std::optional<Value>
+getConstTensorUtil(PatternRewriter &rewriter, Operation *op, ArrayRef<T> vec,
+                   ArrayRef<int64_t> shape, RankedTensorType type);
+
+template <typename T>
```

Comment:
I prefer these have unit tests too, but OK to do in a later PR.

Reply:
Took a TODO to add this (cc: @navahgar).